README.md

```diff
@@ -3,31 +3,112 @@
-- [Tutorial: Chat with PDF](https://github.com/microsoft/promptflow/blob/main/examples/tutorials/e2e-development/chat-with-pdf.md): An end-to-end tutorial on how to build a high quality chat application with prompt flow, including flow development and evaluation with metrics.
+## Installation
 
-Contribute to Prompt flow: please start with our dev setup guide: [dev_setup.md](./docs/dev/dev_setup.md).
+Ensure you have a python environment, `python=3.9` is recommended.
+
+```sh
+pip install promptflow promptflow-tools
+```
+
+## Quick Start ⚡
+
+**Create a chatbot with prompt flow**
+
+Run the command to initiate a prompt flow from a chat template, it creates folder named `my_chatbot` and generates required files within it:
+
+```sh
+pf flow init --flow ./my_chatbot --type chat
+```
+
+**Setup a connection for your API key**
+
+> <details>
+> <summary>For Azure OpenAI key, establish the connection by running:</summary>
+>
+> ```sh
+> # Override keys and endpoint with --set to avoid yaml file changes
+> pf connection create --file ./my_chatbot/azure_openai.yaml --set api_key=<your_api_key> api_base=<your_api_endpoint>
+> ```
+> </details>
+
+For OpenAI key, establish a connection by running the command, using the `openai.yaml` file in the `my_chatbot` folder, which stores your OpenAI key:
+
+```sh
+# Override keys with --set to avoid yaml file changes
+pf connection create --file ./my_chatbot/openai.yaml --set api_key=<your_api_key>
+```
+
+**Chat with your flow**
+
+In the `my_chatbot` folder, there's a `flow.dag.yaml` file that outlines the flow, including inputs/outputs, nodes,  connection, and the LLM model, etc
+
+> Note that in the `chat` node, we're using a connection named `open_ai_connection` and the `gpt-35-turbo` model.
+
+Interact with your chatbot by running: (press `Ctrl + C` to end the session)
+
+```sh
+pf flow test --flow ./my_chatbot --interactive
+```
+
+Next Step! Continue with the **Tutorial**  👇 section to delve deeper into Prompt flow.
+
+## Tutorial 🏃‍♂️
+
+Prompt Flow is a tool designed to **build high quality LLM apps**, the development process in prompt flow follows these steps: develop a flow, improve the flow quality, deploy the flow to production.
+
+### Develop your own LLM apps
+
+#### VS Code Extension<img src="examples/tutorials/quick-start/media/logo_pf.png" alt="logo" width="25"/> 
+
+We also offer a VS Code extension (a flow designer) for an interactive flow development experience with UI. 
+
+<img src="examples/tutorials/quick-start/media/vsc.png" alt="vsc" width="1000"/>
+
+You can install it from the <a href="https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow">visualstudio marketplace</a>.
+
+#### Deep dive into flow development
+
+[Getting Started with Prompt Flow](https://microsoft.github.io/promptflow/how-to-guides/quick-start.html): A step by step guidance to invoke your first flow run.
+
+### Learn from Use Cases
```

Comment:
note other place to change capitalization

Reply:
Changed, 
but this is a title, be sure it needs to be lowercase?